### PR TITLE
#1619 - Add an_element for polyhedron

### LINKED ISSUE #1619: Problems with an_element for rational HPolyhedron
```julia
julia> an_element(HPolyhedron{Rational{Int}}())  # empty list of constraints crashes
ERROR: ArgumentError: An index is out of bound.

julia> an_element(HPolyhedron([HalfSpace([1//1], 0//1)]))  # numeric type changes
1-element Array{Float64,1}:
 0.0
```

See also #1105.
The second point should be ignored (see #1711).

### PR DESCRIPTION
**Pull request by mforets**
Closes #1619.

### CODE REVIEW

File: src/Interfaces/AbstractPolyhedron_functions.jl [hunk 1]
```diff
@@ -578,3 +579,30 @@ function chebyshev_center(P::AbstractPolyhedron{N};
...
+
+An element of a polyhedron.
+
+### Algorithm
+
+An element of the polyhedron is obtained by evaluating its support vector along
+direction ``[1, 0, …, 0]``.
+"""
+function an_element(P::AbstractPolyhedron{N}) where {N<:Real}
+    if dim(P) == -1
```
Comment (schillic) [inline src/Interfaces/AbstractPolyhedron_functions.jl]: Maybe store `dim(P)`?

Comment (mforets) [reply src/Interfaces/AbstractPolyhedron_functions.jl]: ok

Comment (mforets) [reply src/Interfaces/AbstractPolyhedron_functions.jl]: (i think there is no difference in performance, just for readability)

**Comment by mforets** [comment]
> Why did you not do #1867 here?

It is unrelated to #1619.

**Comment by schillic** [comment]
> It is unrelated to #1619.

I do not see why you are not allowed to implement an *efficient* fix instead of just any fix.

**Comment by mforets** [comment]
Keeping `sparsevec` in this branch is the current behavior for all polyhedra. So i take less risks :p . In #1619 i will propose to do both changes as once (this and the generic `an_element`).

File: src/Interfaces/AbstractPolyhedron_functions.jl [hunk 2]
```diff
@@ -578,3 +579,31 @@ function chebyshev_center(P::AbstractPolyhedron{N};
...
 end
+
+"""
+    an_element(P::AbstractPolyhedron{N}) where {N<:Real}
+
+Return some element of a convex set.
+
+### Input
+
+- `S` -- convex set
```
Comment (schillic) [inline src/Interfaces/AbstractPolyhedron_functions.jl]:
```suggestion
- `P` -- polyhedron
```

File: src/Interfaces/AbstractPolyhedron_functions.jl [hunk 3]
```diff
@@ -578,3 +579,31 @@ function chebyshev_center(P::AbstractPolyhedron{N};
...
+### Output
+
+An element of a polyhedron.
+
+### Algorithm
+
+An element of the polyhedron is obtained by evaluating its support vector along
+direction ``[1, 0, …, 0]``.
+"""
+function an_element(P::AbstractPolyhedron{N}) where {N<:Real}
```
Comment (schillic) [inline src/Interfaces/AbstractPolyhedron_functions.jl]: This requires a new entry in the docs.

```
┌ Warning: 1 docstring potentially missing:
│ 
│     LazySets.an_element :: Union{Tuple{AbstractPolyhedron{N}}, Tuple{N}} where N<:Real
```

**Pull request merged**